Orange/widgets/evaluate/owparameterfitter.py

```diff
@@ -223,15 +224,15 @@ def help_event(self, ev: QGraphicsSceneHelpEvent) -> bool:
         text = ""
         if index is not None:
             _, scores = self.__data[index]
-            text = f"<table align=left>" \
-                   f"<tr>" \
-                   f"<td><b>Train:</b></td>" \
+            text = "<table align=left>" \
```

Comment:
I didn't see the lint output, but it usually complains about unnecessary f-strings.